packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -516,4 +516,60 @@ describe('masternode', () => {
       expect(data.length).toBe(64)
     })
   })
+
+  describe('accountHistoryCount', () => {
+    it('should get accountHistoryCount', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.accountHistoryCount()
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThan(0)
+      })
+    })
+
+    it('should get accountHistoryCount with owner as all', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.accountHistoryCount('all')
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThan(0)
+      })
+    })
+
+    it('should get accountHistoryCount with no_rewards option', async () => {
```

Comment:
I think one way we can catch regression issues from node side here is to check if `no_rewards` param does it job by reducing the total count of account history. We can add an additional test case here to call an initial accountHistoryCount with rewards, then call accountHistoryCount with no_rewards. 

Should have less result with no_rewards. The only part we need to add is to make sure that account history has rewards. Maybe by creating a liquidity pool first. Just a suggestion.

Reply:
I'll let @fuxingloh decide on this. Otherwise you can resolve and merge PR if it's ok @siradji

Reply:
I would love to try and implement your suggestion but i don't know and i have a lot of other implementations to work on. But if @fuxingloh wants it integrated, i will do it.

Reply:
yeah, we should definetly do it. 

it might be complex going into the defi system but it's good to start learning now too.

Reply:
I will definitely do that.